src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    private static final FigureSupplire figureSupplire = new FigureSupplire();
+
+    public static void main(String[] args) {
+        Figure[] figure = new Figure[6];
+        for (int i = 0; i < figure.length; i++) {
+            if (i < figure.length / 2) {
+                figure[i] = figureSupplire.getRandomFigure();
+            } else {
+                figure[i] = figureSupplire.getDefaultFigure();
+            }
+            figure[i].getDraw();

```

Comment:
The method name 'getDraw' might be confusing. Consider renaming it to 'draw' or 'display' to better convey its purpose. 🎨